### PR TITLE
DNSIMPLE: do not support unpaired double quotes in TXT

### PR DESCRIPTION
**Pull request by onlyhavecans**
DNSimple supports multiple double-quotes strings in a TXT record, but does not correctly support unpaired or escaped double-quotes currently.

IE the following are valid
```
asdf
"asdf"
"asdf" "asdf"
!@#$ %^&*()([][{}{<></'`:;-_=+\
```

however `as\"df` and `as"df` are not

This removes the extra string processing in getTargetRecordPriority as all tests pass without it and currently only double-quotes cause problems in our TXT validations.

I added another test to prove additional quoting is not needed. We can remove it if undesired.

Also applied small lint changes to make my editor happy.

### CODE REVIEW

**Comment by tlimoncelli** [comment]
Looking good so far!  Just some suggestions.

Tom

**Comment by onlyhavecans** [comment]
Awesome to hear! However, I cannot see your suggestions. Maybe they are not committed? Just let me know what I can do to improve the pr. thanks 👏🏻

**Comment by tlimoncelli** [comment]
Right you are!  I didn't click the last submit.  D'oh!

File: integrationTest/integration_test.go [hunk 1]
```diff
@@ -905,6 +905,8 @@ func makeTests(t *testing.T) []*TestGroup {
 			tc("Create TXT with double-quote", txt("foodq", `quo"te`)),
 			clear(),
 			tc("Create TXT with ws at end", txt("foows1", "with space at end ")),
+			clear(),
+			tc("Create TXT with frequently escaped characters", txt("fooex", "!^.*$@#%^&()([][{}{<></:;-_=+\\")),
```
[the remark this answers is not in the document]
Comment (onlyhavecans) [reply integrationTest/integration_test.go]: Thanks for the feedback! done

File: providers/dnsimple/dnsimpleProvider.go [hunk 2]
```diff
@@ -583,21 +583,13 @@ func getTargetRecordContent(rc *models.RecordConfig) string {
...
 	case "SRV":
 		return fmt.Sprintf("%d %d %s", rc.SrvWeight, rc.SrvPort, rc.GetTargetField())
-	case "TXT":
-		quoted := make([]string, len(rc.TxtStrings))
-		for i := range rc.TxtStrings {
-			quoted[i] = quoteDNSString(rc.TxtStrings[i])
-		}
-		return strings.Join(quoted, " ")
 	default:
 		return rc.GetTargetField()
```
[the remark this answers is not in the document]
Comment (onlyhavecans) [reply providers/dnsimple/dnsimpleProvider.go:597]: I tested `rc.GetTargetRFC1035Quoted()` against both integration tests and used my personal configuration to make several changes and found it does the correct thing on all the updates I gave it so let's go with that route. Thanks for the feedback

**Pull request merged**